config/kubernetes/sig-contributor-experience/teams.yaml

```diff
@@ -88,51 +78,8 @@ teams:
     - justaugustus
     - lachie83
     - LappleApple
-    - mattfarina
-    privacy: closed
-  sig-contributor-experience:
-    description: Improve contributor productivity
-    maintainers:
-    - cblecker
-    - idvoretskyi
-    - mrbobbytables
-    - nikhita
-    members:
-    - alisondy
-    - castrojo
-    - dims
-    - eparis
-    - foxish
-    - grodrigues3
-    - hongchaodeng
-    - idealhack
-    - jessfraz
-    - Kashomon
-    - lavalamp
-    - monopole
-    - parispittman
-    - philips
-    - Phillels
-    - pigmej
-    - pwittrock
-    - thockin
-    - xiang90
     privacy: closed
-    previously:
-      - sig-contributor-experience-misc-use-only-as-a-last-resort
     teams:
```

Comment:
This messes up the sub teams, doesn't it?

Reply:
You'r right it does -_- could move them up a level, on a side note - it was not caught by our tests :x